docs/DeveloperGuide.md

```diff
@@ -65,6 +65,36 @@ The Parser component:
 We pass in the`itemList`to the`Command`object instead of using a global variable to ease testing. This way, we can 
 add, delete and update`Expenditure`entries in a new`itemList`during testing without affecting the actual`itemList`.
 
+### Command component 
+The source can be found in [command](https://github.com/AY2122S2-CS2113T-T10-4/tp/blob/master/src/main/java/seedu/mindmymoney/command)
+
+![command_diagram](images/command_diagram.png)
+<br/> Fig 5 - Command Diagram
+
+The Command component consists of `Command` abstract class, `AddCommand`, `ByeCommand`, `DeleteCommand`, `HelpCommand`
+, `ListCommand`and `UpdateCommand` class.
+
+The Command component:
+- `Command` class is an abstract class and `AddCommand`, `ByeCommand`, `DeleteCommand`, `HelpCommand`, `ListCommand`
+and `UpdateCommand`extends the `Command` class.
+- Parser.parseCommand() creates the different kinds of command that can be executed. Only 1 command object can be 
```

Comment:
Maybe can highlight Parser.parseCommand() since it's part of our code. Also some rephrasing to suit the starting prompt "The command component:"
```suggestion
- Provides all the Command classes which can be instantiated by `Parser.parseCommand()`. The Command objects can then be executed. Only 1 Command object can be 
```

Reply:
fixed